src/python/WMCore/Lexicon.py

```diff
@@ -702,6 +702,15 @@ def primaryDatasetType(candidate):
     raise AssertionError("Invalid primary dataset type : %s should be 'mc' or 'data' or 'test'" % candidate)
 
 
+def subRequestType(candidate):
+    subTypes = ["MC", "ReDigi", "Pilot", "RelVal", "HIRelVal", "ReReco", ""]
```

Comment:
Does this set of subRequestTypes consider also all the possible (if any) sub types for T0?

Reply:
AFAIK T0 does not use it. If they do, we can then extend it in the future.